From b738a596d64643aea279531e2bbf4baa8445cc58 Mon Sep 17 00:00:00 2001
From: mete <mete@26e1f6f6-46e4-0310-a0b7-a8a415fd4c45>
Date: Wed, 25 May 2011 23:32:15 +0000
Subject: [PATCH 08/10] Change default partitioning method as non selected

Not to select any option as default to user place.

git-svn-id: https://svn.pardus.org.tr/uludag/trunk/yali@36991 26e1f6f6-46e4-0310-a0b7-a8a415fd4c45
---
 yali/gui/ScrPartitionAuto.py  |    6 +++---
 yali/gui/Ui/autopartwidget.ui |    4 ++--
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/yali/gui/ScrPartitionAuto.py b/yali/gui/ScrPartitionAuto.py
index c55ab65..0e464fa 100644
--- a/yali/gui/ScrPartitionAuto.py
+++ b/yali/gui/ScrPartitionAuto.py
@@ -50,14 +50,14 @@ class Widget(QWidget, ScreenWidget):
             ctx.mainScreen.enableNext()
 
     def setPartitioningType(self):
-        if self.storage.clearPartType is None:
-            self.ui.autopartType.setCurrentRow(CUSTOM)
-        elif self.storage.clearPartType == CLEARPART_TYPE_NONE:
+        if self.storage.clearPartType == CLEARPART_TYPE_NONE:
             self.ui.autopartType.setCurrentRow(FREE_SPACE)
         elif self.storage.clearPartType == CLEARPART_TYPE_ALL:
             self.ui.autopartType.setCurrentRow(USE_ALL_SPACE)
 
     def shown(self):
+        if len(self.storage.clearPartDisks) > 1:
+            self.ui.autopartType.item(USE_ALL_SPACE).setText(_("Use All Disks"))
         self.setPartitioningType()
 
     def execute(self):
diff --git a/yali/gui/Ui/autopartwidget.ui b/yali/gui/Ui/autopartwidget.ui
index 0a90b61..39b8025 100644
--- a/yali/gui/Ui/autopartwidget.ui
+++ b/yali/gui/Ui/autopartwidget.ui
@@ -208,7 +208,7 @@ text-decoration: normal;
           </property>
           <item>
            <property name="text">
-            <string>Use All Space</string>
+            <string>Use All Disk</string>
            </property>
            <property name="icon">
             <iconset>
@@ -235,7 +235,7 @@ text-decoration: normal;
           </item>
           <item>
            <property name="text">
-            <string>Create Custom Layout</string>
+            <string>Manual Partitioning</string>
            </property>
            <property name="icon">
             <iconset>
-- 
1.7.5

